datagateway_api/src/api_start_utils.py

```diff
@@ -283,13 +289,14 @@ def create_api_endpoints(flask_app, api, specs):
     # Search API endpoints
     if Config.config.search_api is not None:
         search_api_spec = next(
-            (spec for spec in specs if spec.title == "Search API"), None,
+            (spec for spec in specs if spec.title == "Search API"),
+            None,
         )
         search_api_extension = Config.config.search_api.extension
         search_api_entity_endpoints = {
-            "datasets": "Dataset",
-            "documents": "Document",
-            "instruments": "Instrument",
+            "Datasets": "Dataset",
+            "Documents": "Document",
+            "Instruments": "Instrument",
```

Comment:
Do you know why this has been changed? The keys are for the endpoints, which are typically lower case

Reply:
I needed to change that in order to match the PANOSC endpoints that use upper case:
https://github.com/panosc-eu/search-api/blob/master/doc/dataset-example-queries.md#curl

Otherwise, the aggregator will return 404

Reply:
Ah ok, I didn't notice that before but I guess that's a required change if the server on PaNOSC's side won't recognise lowercase endpoint names.

Reply:
Exact.